tests/x11/awesome_xterm.pm

```diff
@@ -0,0 +1,19 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    mouse_hide(1);
```

Comment:
why the mouse_hide here? the mouse should be pretty much hidden in every test before.